lib/docx/body-reader.js

```diff
@@ -133,14 +134,27 @@ function BodyReader(options) {
             return readXmlElements(element.children)
                 .map(function(children) {
                     var properties = _.find(children, isRunProperties);
-                    
-                    return new documents.Run(
-                        children.filter(negate(isRunProperties)),
-                        properties
-                    );
+                    var filteredChildren = children.filter(negate(isRunProperties));
+
+                    if (properties && properties.styleName === 'Hyperlink' && lastInstrText) {
```

Comment:
Can you not use the fact that the text of `w:instrText` starts with `HYPERLINK`? Relying on the style at all feels brittle to me.